### DISCUSSION TITLE
Caching system to speed up track reload

### DISCUSSION THREAD

**Discussion by weber8thomas** [GMOD/jbrowse-components#4203] [Q&A]
Hi, I wanted to know if some existing caching mechanisms to speed up track loading that were already been selected ? If not, what would be the different strategies to do this? 
Many thanks in advance!

**Comment by cmdcolin** [comment]
@weber8thomas if you have more details on your scenario, I might be able to advise. but, we are aware of various slowness/perf issues in the app...various github issues keep track of them. we are actively brainstorming them but happy to advise if you can describe your scenario more

**Reply by weber8thomas** [reply]
Related to  https://github.com/GMOD/jbrowse-components/discussions/4153 , I'd like to dynamically add/remove tracks based on the user interaction with other components present on my platform. As I'm onboarding jbrowse into a iframe container, one current limitation is that I need to reload that iframe when there are major changes on my platform, leading to a complete reloading of all the tracks.
